tests/foreman/cli/test_contentaccess.py

```diff
@@ -38,42 +41,48 @@
 
 
 @pytest.fixture(scope='module')
-def module_lce(module_sca_manifest_org):
-    return entities.LifecycleEnvironment(organization=module_sca_manifest_org).create()
-
+def rh_repo_setup_ak(module_sca_manifest_org, module_target_sat):
+    """Use module sca manifest org, creates rhst repo & syncs it,
+    also create CV, LCE & AK and return AK"""
+    rh_repo_id = module_target_sat.api_factory.enable_rhrepo_and_fetchid(
+        basearch=DEFAULT_ARCHITECTURE,
+        org_id=module_sca_manifest_org.id,
+        product=PRDS['rhel'],
+        repo=REPOS['rhst7']['name'],
+        reposet=REPOSET['rhst7'],
+        releasever=None,
+    )
+    # Sync step because repo is not synced by default
+    rh_repo = entities.Repository(id=rh_repo_id).read()
+    rh_repo.sync()
 
-@pytest.fixture(scope="module")
-def rh_repo_cv(module_sca_manifest_org, rh_repo_gt_manifest, module_lce):
-    rh_repo_cv = entities.ContentView(organization=module_sca_manifest_org).create()
+    # Create CV, LCE and AK
+    cv = entities.ContentView(organization=module_sca_manifest_org).create()
```

Comment:
For this and other entity create/read, could I recommend updating to use `target_sat.api.Entity().read() .create()` ? As you are now using `module_target_sat` fixture.
```suggestion
    cv = module_target_sat.api.ContentView(
         organization=module_sca_manifest_org
    ).create()
```

Reply:
Sorry I always forget this change

Reply:
no worries same here! 
I also just noticed we could use cli create() and read(), 
if you prefer for cli test. Non-blocking!